components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -141,21 +141,40 @@ impl TransitionProperty {
 
     /// Parse a transition-property value.
     pub fn parse<'i, 't>(input: &mut Parser<'i, 't>) -> Result<Self, ParseError<'i>> {
+        // FIXME(https://github.com/rust-lang/rust/issues/33156): remove this
+        // enum and use PropertyId when stable Rust allows destructors in
+        // statics.
+        pub enum StaticId {
+            All,
```

Comment:
Hmm, I suspect it's a bug that we don't do aliases here, and fixing it will allow to use the same `StaticId` as `PropertyId::parse`. @hiikezoe @birtles, is that right?

Reply:
Yeah, we need to handle aliases for transition property too.  It seems to me, unfortunately, we don't have such test cases in test_transitions_per_property.html though.

I am not sure we can reuse PropertyId::parse here, the code has been changed last time I see.   Now I don't quite understand where 'display' property is handled as an exception for transition-property,  and discrete type properties.

Reply:
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1437412 for this.